### PR TITLE
Take Seths new change to fix ReplayActivity 

### PR DESCRIPTION
**Pull request by kmadsen**
## Description
EDIT: Talked to @cafesilencio and he was working on another pr that keeps us moving forward. Typically revert breaking changes but this was simple enough to fix. Taking the ReplayActivity changes from here https://github.com/mapbox/mapbox-navigation-android/pull/2781

ReplayActivity doesn't work for a few reasons. It opens in the ocean, if you fix that it won't replay the route. I was digging into fixing it, but found that reverting the ReplayActivity changes from here fix it https://github.com/mapbox/mapbox-navigation-android/pull/2783

- [ ] I have added any issue links
- [ ] I have added all related labels (`bug`, `feature`, `new API(s)`, `SEMVER`, etc.)
- [ ] I have added the appropriate milestone and project boards

## Testing

Please describe the manual tests that you ran to verify your changes

- [ ] I have tested locally (including `SNAPSHOT` upstream dependencies if needed) through testapp/demo app and run all activities to avoid regressions
- [x] I have tested via a test drive, or a simulation/mock location app
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have updated the `CHANGELOG` including this PR

### CODE REVIEW

File: examples/src/main/java/com/mapbox/navigation/examples/core/ReplayActivity.kt [hunk 1]
```diff
@@ -95,6 +102,25 @@ class ReplayActivity : AppCompatActivity(), OnMapReadyCallback {
...
+        locationComponent = mapboxMap.locationComponent
+    }
+
+    @SuppressLint("RestrictedApi")
+    fun initLocationComponent(loadedMapStyle: Style, mapboxMap: MapboxMap) {
+        mapboxMap.moveCamera(CameraUpdateFactory.zoomTo(15.0))
+        mapboxMap.locationComponent.let { locationComponent ->
+            val locationComponentActivationOptions =
+                LocationComponentActivationOptions.builder(this, loadedMapStyle)
+                    // .useDefaultLocationEngine(true)
```
Comment (Guardiola31337) [inline examples/src/main/java/com/mapbox/navigation/examples/core/ReplayActivity.kt:114]: NIT Remove commented out code?

Comment (kmadsen) [reply examples/src/main/java/com/mapbox/navigation/examples/core/ReplayActivity.kt:114]: i'm working on ReplayActivity and am removing this. going to leave this PR as a pure revert so there's less conflicts

Comment (kmadsen) [reply examples/src/main/java/com/mapbox/navigation/examples/core/ReplayActivity.kt]: ah ok removed it's fine lol

**Comment by kmadsen** [comment]
Tested and this change would fix it https://github.com/mapbox/mapbox-navigation-android/pull/2781

**Pull request merged**